src/rsz/src/Resizer.tcl

```diff
@@ -156,12 +156,6 @@ proc set_wire_rc { args } {
       set wire_res [rsz::layer_resistance $layer $corner]
       set wire_cap [rsz::layer_capacitance $layer $corner]
     }
-
-    # Unfortunately this does not work very well with technologies like sky130
-    # that use inappropriate kohm/pf units.
-    if { 0 } {
-      utl::info RSZ 61 "$signal_clk wire resistance [sta::format_resistance [expr $wire_res * 1e-6] 6] [sta::unit_scale_abbreviation resistance][sta::unit_suffix resistance]/um capacitance [sta::format_capacitance [expr $wire_cap * 1e-6] 6] [sta::unit_scale_abbreviation capacitance][sta::unit_suffix capacitance]/um."
-    }
```

Comment:
`tclint` flagged this line for a length violation, I made the call to delete it wholesale myself (since it's dead code). If you'd prefer to keep it, I can change the commit to just fix the line length violation instead.